src/server.js

```diff
@@ -0,0 +1,7 @@
+const express = require('express');
+const app = require("./app.js");
+
+app.set("PORT", 6002)
+app.listen(app.get("PORT"), ()=>{
+  console.log(`The Server Is Up And Running`);
```

Comment:
For longer projects, it's great to include a link here to your local host. e.g. `The server is running on http://localhost:6002`. Double check this works before pushing.